<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWindow</class>
 <widget class="QMainWindow" name="LobbyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LobbyWindow</string>
  </property>
  <widget class="QStackedWidget" name="centralWidget">
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="firstLobbyScene">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>351</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="playMatchButton">
        <property name="styleSheet">
         <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
        </property>
        <property name="text">
         <string>PLAY MATCH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createMapButton">
        <property name="styleSheet">
         <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
        </property>
        <property name="text">
         <string>CREATE MAP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="loginScene">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>471</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="loginLayout">
      <item>
       <widget class="QLineEdit" name="playerNameLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Player name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverPortLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Server port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostNameLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Host name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="exampleHostLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 124);
font: 700 10pt &quot;Ubuntu Mono&quot;;</string>
        </property>
        <property name="text">
         <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;(e.g. localhost)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>340</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
     </property>
     <property name="text">
      <string>CONFIRM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="joinMatchButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
     </property>
     <property name="text">
      <string>JOIN MATCH</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createMatchButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>440</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
     </property>
     <property name="text">
      <string>CREATE MATCH</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="joinMatchScene">
    <widget class="QComboBox" name="matchList">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>260</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Available matches</string>
     </property>
    </widget>
    <widget class="QPushButton" name="joinButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>350</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
     </property>
     <property name="text">
      <string>JOIN</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createMatchScene">
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>240</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="createMatchLayout">
      <item>
       <widget class="QLineEdit" name="matchNameLineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Match name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numberLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Number of players</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numberConditionLabel">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 124);
font: 700 10pt &quot;Ubuntu Mono&quot;;</string>
        </property>
        <property name="text">
         <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;(min 2 players)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>410</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 15pt &quot;Ubuntu Mono&quot;;</string>
     </property>
     <property name="text">
      <string>CREATE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="mapList">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>350</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Available maps</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
